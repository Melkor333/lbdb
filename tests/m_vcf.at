dnl Copyright (c) 2024 Roland Rosenfeld <roland@spinnaker.de>
dnl
dnl This program is free software; you can redistribute it and/or modify
dnl it under the terms of the GNU General Public License as published by
dnl the Free Software Foundation; either version 2 of the License, or
dnl (at your option) any later version.
dnl
dnl This program is distributed in the hope that it will be useful,
dnl but WITHOUT ANY WARRANTY; without even the implied warranty of
dnl MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
dnl GNU General Public License for more details.
dnl
dnl You should have received a copy of the GNU General Public License
dnl along with this program; if not, write to the Free Software Foundation,
dnl Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.

AT_BANNER([Test m_vcf])

AT_SETUP(m_vcf)

AT_CHECK([cat $abs_srcdir/data/m_vcf.expected > expout])
AT_CHECK([cat $abs_srcdir/data/example.vcf > example.vcf])
AT_CHECK([env LBDB_OVERRIDE_METHODS=m_vcf \
              OVERRIDE_VCF_FILES=example.vcf \
          lbdbq muster], [], [expout])

AT_CLEANUP
