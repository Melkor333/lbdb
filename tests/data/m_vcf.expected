lbdbq: 3 matches
erika.home@example.com	Erika Musterfrau	(VCF)
erika.musterfrau@example.com	Erika Musterfrau	(VCF)
max.mustermann@example.com	Max Mustermann	(VCF)
